sdk/examples/wallet/07_send_native_tokens.rs

```diff
@@ -15,40 +20,51 @@ use iota_sdk::{
 };
 use primitive_types::U256;
 
+// The account alias used in this example
+const ACCOUNT_ALIAS: &str = "Alice";
+// The native token amount to send
+const SEND_NATIVE_TOKEN_AMOUNT: u64 = 10;
+// The address to send the tokens to
+const RECV_ADDRESS: &str = "rms1qpszqzadsym6wpppd6z037dvlejmjuke7s24hm95s9fg9vpua7vluaw60xu";
+// The wallet database folder
+const WALLET_DB_PATH: &str = "./example.walletdb";
```

Comment:
Same here, looks annoying to change, maybe make it a env var?

Reply:
I agree :+1:

Reply:
Done